src/seedu/addressbook/commands/Command.java

```diff
@@ -70,4 +70,14 @@ public int getTargetIndex() {
     public void setTargetIndex(int targetIndex) {
         this.targetIndex = targetIndex;
     }
+
+    /**
+     * Checks if command mutates data in the address book.
+     *
+     * @return true is command mutates data
```

Comment:
These 2 lines could be shrunk into 1 sentence that describes what isMutating does. In this case, you won't need the return since they would be the same thing.

Reply:
This means "Checks if command mutates data in the address book." is sufficient as a description in this case?